TelemetrySite/server/utils.py

```diff
@@ -5,25 +5,27 @@
 ## Use to connect to the database using the .env file
 def connect():
     return psycopg2.connect(
-                            database = "bert", 
-                            user     = os.getenv("DB_USER"), 
-                            password = os.getenv("PASSWORD"), 
-                            host     = os.getenv("HOST"), 
-                            port     = os.getenv("PORT")
-                            )
+        database="bert",
```

Comment:
@tmb7852 At some point, it would be pretty cool if the database was not named `bert`